mail_factory/tests/test_views.py

```diff
@@ -21,6 +20,12 @@
 from ..forms import MailForm
 
 
+try:
+    from django.urls import reverse
+except ImportError:
+    from django.core.urlresolvers import reverse
```

Comment:
Can you add a comment telling for which versions this try except is so that when we remove support we know we can remove this too.

Reply:
Sure!